README.md

```diff
@@ -4,9 +4,23 @@
 [<img src="https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-parser-combinators_2.13.svg?label=latest%20release%20for%202.13"/>](http://search.maven.org/#search%7Cga%7C1%7Cg%3Aorg.scala-lang.modules%20a%3Ascala-parser-combinators_2.13)
 [<img src="https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-parser-combinators_3.svg?label=latest%20release%20for%203"/>](http://search.maven.org/#search%7Cga%7C1%7Cg%3Aorg.scala-lang.modules%20a%3Ascala-parser-combinators_3)
 
-### Scala Standard Parser Combinator Library
+This was originally part of the Scala standard library, but is now community-maintained, under the guidance of the Scala team at Lightbend. If you are interested in joining the maintainers team, please contact [@Philippus](https://github.com/philippus) or [@SethTisue](https://github.com/SethTisue).
 
-This library was originally part of the Scala standard library, but is now community-maintained, under the guidance of the Scala team at Lightbend. If you are interested in helping please contact [@Philippus](https://github.com/philippus) or [@SethTisue](https://github.com/SethTisue).
+## Choosing a parsing library
+
+This library's main strengths are:
+
+* Stability. It's been around and in wide use for more than a decade.
+* The codebase is modest in size and its internals are fairly simple.
+* It's plain vanilla Scala. No macros, code generation, or other magic is involved.
```

Comment:
I'd probably add that there seem to be versions of this library for all versions of scala (including js and native), while several of the alternatives can't make that claim.